commitizen/tags.py

```diff
@@ -146,7 +146,8 @@ def extract_version(self, tag: GitTag) -> Version:
             m for regex in self.version_regexes if (m := regex.fullmatch(tag.name))
         )
         if not (m := next(candidates, None)):
-            raise InvalidVersion()
+            regexes = ", ".join([str(r) for r in self.version_regexes])
+            raise InvalidVersion(f"Invalid version tag: '{tag.name}' does not match any configured tag format: {regexes}")
```

Comment:
Maybe something shorter, without the regexp:
```suggestion
            raise InvalidVersion(f"Invalid version tag: '{tag.name}' does not match any configured tag format")
```

With many regexps or when you have many tags that don't match (or worse if you have both many patterns and many unmatched tags), it will break the terminal output making it harder to read and defeating the purpose of this PR.